components/brave_vpn/resources/panel/components/flag/index.tsx

```diff
@@ -24,25 +24,21 @@ export const IconBox = styled.span`
 const COUNTRIES = ['GB', 'US', 'CH', 'ES', 'SG', 'NL', 'JP', 'DE', 'FR', 'CA', 'AU']
 
 function Flag (props: Props) {
-  let [url, setUrl] = React.useState(undefined)
+  let [url, setUrl] = React.useState('')
 
   React.useEffect(() => {
     let canceled = false
```

Comment:
We can get rid of `canceled` variable completely. This was useful for async operations.